pulsar-broker/src/main/java/org/apache/pulsar/broker/service/nonpersistent/NonPersistentSubscription.java

```diff
@@ -210,7 +210,7 @@ public void consumerFlow(Consumer consumer, int additionalNumberOfMessages) {
 
     @Override
     public void acknowledgeMessage(List<Position> position, AckType ackType, Map<String, Long> properties) {
-        // No-op
+        updateLastActive();
```

Comment:
Please help add a test to avoid the regression.

Reply:
I will add a test to cover this.

Reply:
I found that non-persistent consumer ack will not sent ack command to broker, so removing the modification in `NonPersistentSubscription.java`.